python/cugraph/dask/bfs/bfs.py

```diff
@@ -0,0 +1,46 @@
+def bfs(edge_list, start, return_distances=False):
```

Comment:
This File is not needed. Corresponding Pagerank counterpart is also going to be removed for 0.15. No point adding it to be soon removed